cognite/client/_api/workflows.py

```diff
@@ -48,6 +54,144 @@ def wrap_workflow_ids(
     return WorkflowIds.load(workflow_version_ids).dump(camel_case=True, as_external_id=True)
 
 
+class WorkflowTriggerAPI(APIClient):
+    _RESOURCE_PATH = "/workflows/triggers"
+
+    def __init__(
+        self,
+        config: ClientConfig,
+        api_version: str | None,
+        cognite_client: CogniteClient,
+    ) -> None:
+        super().__init__(config, api_version, cognite_client)
+        self._api_subversion = "beta"
+        self._warning = FeaturePreviewWarning(
+            api_maturity="beta", sdk_maturity="beta", feature_name="Workflow Orchestration"
+        )
+
+    def create(
+        self,
+        workflow_trigger: WorkflowTriggerCreate,
+        client_credentials: ClientCredentials | dict | None = None,
+    ) -> WorkflowTrigger:
+        """`Create a new trigger for a workflow. <https://api-docs.cognite.com/20230101/tag/Workflow-triggers/operation/createTriggers>`_
+
+        Args:
+            workflow_trigger (WorkflowTriggerCreate): The workflow trigger specitification.
+            client_credentials (ClientCredentials | dict | None): Specific credentials that should be used to trigger the workflow execution. When passed will take precedence over the current credentials.
+
+        Returns:
+            WorkflowTrigger: The created workflow trigger specification.
+
+        Examples:
+
+            Create a new scheduled trigger for a workflow:
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes.workflows import WorkflowTriggerCreate, WorkflowScheduledTriggerRule
+                >>> client = CogniteClient()
+                >>> client.workflows.triggers.create(
+                ...     WorkflowTriggerCreate(
+                ...         external_id="my_trigger",
+                ...         trigger_rule=WorkflowScheduledTriggerRule(cron_expression="0 0 * * *"),
+                ...         workflow_external_id="my_workflow",
+                ...         workflow_version="1",
+                ...         input={"a": 1, "b": 2},
+                ...     )
+                ... )
+        """
+        nonce = create_session_and_return_nonce(
```

Comment:
Creating the warning is not enough :) 
```suggestion
        """
        self._warning.warn()
        nonce = create_session_and_return_nonce(
```